packages/amplify-nodejs-function-runtime-provider/src/__tests__/utils/invoke.test.ts

```diff
@@ -1,5 +1,6 @@
 import * as path from 'path';
 import { invoke } from '../../utils/invoke';
+jest.setTimeout(10000);
```

Comment:
What is the purpose of this?

Reply:
added this since the tests were failing, running timeout

Reply:
Looks like the failing tests are not related to the change below. Are these two tests also failing on the latest dev?